rubygem-hammer_cli_gutterball.spec

```diff
@@ -50,20 +48,21 @@ gem install --local --install-dir .%{gem_dir} \
 
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d
-install -m 755 %{SOURCE1} %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
+install -m 755 .%{gem_instdir}/config/gutterball.yml %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
+
 mkdir -p %{buildroot}%{gem_dir}
 cp -pa .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %files
-%dir %{geminstdir}
-%{geminstdir}/
-%config(noreplace) %{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
-%exclude %{gem_dir}/cache/%{gemname}-%{version}.gem
-%{gem_dir}/specifications/%{gemname}-%{version}.gemspec
+%dir %{gem_instdir}
+%{gem_instdir}/
+%exclude %{gem_dir}/cache/%{gem_name}-%{version}.gem
+%{gem_dir}/specifications/%{gem_name}-%{version}.gemspec
 
 %files doc
-%doc %{gem_dir}/doc/%{gemname}-%{version}
+%doc %{gem_dir}/doc/%{gem_name}-%{version}
+%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
```

Comment:
The foreman hammer cli specs handle it this way -- https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rubygem-hammer_cli_foreman/rubygem-hammer_cli_foreman.spec#L71